### ISSUE TITLE
Loch issue on MacOS 10.13.6

### ISSUE THREAD

**Issue by martinsluka** [therion/therion#461]
"Open Loch and open file" or "drag file to loch" or "open file in loch" work well. But double-click to .lox file produce next error:

Process:               loch [6706]
Path:                  /private/var/folders/*/loch.app/Contents/MacOS/loch
Identifier:            org.wxwindows..
Version:               1.0 (1.0)
Code Type:             X86-64 (Native)
Parent Process:        ??? [1]
Responsible:           loch [6706]
User ID:               501

Date/Time:             2022-11-04 14:13:56.673 +0100
OS Version:            Mac OS X 10.13.6 (17G14042)
Report Version:        12
Anonymous UUID:        EF172A66-4FEF-A7EF-6F19-33F2C5C7C8AC

Time Awake Since Boot: 71000 seconds

System Integrity Protection: enabled

Notes:                 Translocated Process

Crashed Thread:        0

Exception Type:        EXC_CRASH (SIGABRT)
Exception Codes:       0x0000000000000000, 0x0000000000000000
Exception Note:        EXC_CORPSE_NOTIFY

Termination Reason:    DYLD, [0x1] Library missing

Application Specific Information:
dyld: launch, loading dependent libraries

Dyld Error Message:
 Library not loaded: /usr/local/opt/wxmac/lib/libwx_osx_cocoau_gl-3.0.dylib
 Referenced from: /private/var/folders/*/loch.app/Contents/MacOS/loch
 Reason: image not found

Binary Images:
      0x109047000 -        0x1090daffb +org.wxwindows.. (1.0 - 1.0) <32B73056-1DD3-3F5C-B16D-3DDAC05CC4D4> /var/folders/*/loch.app/Contents/MacOS/loch
      0x112101000 -        0x11214badf  dyld (551.5) <CB9BFB56-4511-36F1-A546-891FF770C01C> /usr/lib/dyld
   0x7fff5b805000 -     0x7fff5b817ffb  libz.1.dylib (70) <47278BDC-596E-3037-B8DA-05FE9D5DBEF6> /usr/lib/libz.1.dylib

Model: iMac10,1, BootROM 215.0.0.0.0, 2 processors, Intel Core 2 Duo, 3,06 GHz, 12 GB, SMC 1.52f9
Graphics: NVIDIA GeForce 9400, NVIDIA GeForce 9400
Memory Module: BANK 0/DIMM0, 2 GB, DDR3, 1067 MHz, 0x80AD, 0x484D54313235533642465238432D47372020
Memory Module: BANK 1/DIMM0, 4 GB, DDR3, 1067 MHz, 0x0198, 0x393930353432382D3034332E413031472020
Memory Module: BANK 0/DIMM1, 2 GB, DDR3, 1067 MHz, 0x80AD, 0x484D54313235533642465238432D47372020
Memory Module: BANK 1/DIMM1, 4 GB, DDR3, 1067 MHz, 0x0198, 0x393930353432382D3034332E413031472020
AirPort: spairport_wireless_card_type_airport_extreme (0x168C, 0x8F), Atheros 9280: 4.0.74.0-P2P
Bluetooth: Version 6.0.7f22, 3 services, 27 devices, 1 incoming serial ports
Network Service: Wi-Fi, AirPort, en1
Serial ATA Device: ST3500418ASQ, 500,11 GB
Serial ATA Device: PIONEER DVD-RW  DVRTS09
USB Device: USB Bus
USB Device: IR Receiver
USB Device: USB Bus
USB Device: Apple Optical USB Mouse
USB Device: BRCM2046 Hub
USB Device: Bluetooth USB Host Controller
USB Device: USB 2.0 Bus
USB Device: Internal Memory Card Reader
USB Device: USB 2.0 Bus
USB Device: Built-in iSight
USB Device: Keyboard Hub
USB Device: Apple Keyboard
Thunderbolt Bus:

**Comment by Afforix** [comment]
> Dyld Error Message:
Library not loaded: /usr/local/opt/wxmac/lib/libwx_osx_cocoau_gl-3.0.dylib
Referenced from: /private/var/folders/*/loch.app/Contents/MacOS/loch
Reason: image not found

What package with `wxWidgets` library do you have installed? There is a deprecated `wxwidgets@3.0` (formerly known as wxmac@3.0), and current version is `wxwidgets@3.2`. Also I don't see `macOS 10.13.6 High Sierra` listed as a supported system for both [wxwidgets@3.2](https://formulae.brew.sh/formula/wxwidgets) and [wxwidgets@3.0](https://formulae.brew.sh/formula/wxwidgets@3.0). High Sierra is unsupported by Apple itself for 2 years now.

**Comment by martinsluka** [comment]
Yes, I know it is quite old comp with old system. If it is only problem, I’ll use not direct way to open lox files. 

Thanks

M. 

Odesláno z iPhonu
